code/modules/telesci/gps.dm

```diff
@@ -79,3 +79,8 @@ var/list/GPS_list = list()
 	icon_state = "gps-m"
 	gpstag = "MINE0"
 	desc = "A positioning system helpful for rescuing trapped or injured miners, keeping one on you at all times while mining might just save your life."
+
+/obj/item/device/gps/internal
+	icon_state = null
+	gpstag = "Eerie Signal"
+	desc = "Report to a coder immediately."
```

Comment:
Should I just put a link to new issue instead? A lot of internal obj use this kind of desc.